src/components/ProductList.tsx

```diff
@@ -27,85 +22,49 @@ export type ProductListProps = {
 }
 
 const ProductList = ({ actionLabel, content, category }: ProductListProps) => {
-  const shadow = useColorModeValue("tableBox.light", "tableBox.dark")
-
   const CATEGORY_NAME = "category-name"
 
   return (
-    <Box boxSize="full">
-      <OldHeading
-        as="h3"
+    <div className="w-full">
+      <h3
         id={CATEGORY_NAME}
-        fontSize="2xl"
-        borderBottom="2px solid"
-        borderColor="border"
-        paddingBottom={4}
-        marginBottom={0}
+        className="mb-0 mt-10 border-b-2 border-border pb-4 text-2xl"
```

Comment:
removing since it is unnecessary, it will default to the `border` color
```suggestion
        className="mb-0 mt-10 border-b-2 pb-4 text-2xl"
```